types/tslint.json

```diff
@@ -18,6 +18,7 @@
     "strict-export-declare-modifiers": false,
     "whitespace": false,
     "use-default-type-parameter": false,
-    "no-redundant-jsdoc": false
+    "no-redundant-jsdoc": false,
+    "no-any-union": false
```

Comment:
For what type do we need this currently?

Reply:
There are two in "globals.d.ts", where at least the websocket one is consistent with the ones TypeScript provides for browser code.

Reply:
We could constrain it a little more, but I think it would make a typing error for anything already assuming it is a specific value without checking.